.teamcity/.mvn/gradle-enterprise.xml

```diff
@@ -0,0 +1,33 @@
+
+
+<!--
+  ~ Copyright 2022 the original author or authors.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~      http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<gradleEnterprise
+        xmlns="https://www.gradle.com/gradle-enterprise-maven" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="https://www.gradle.com/gradle-enterprise-maven https://www.gradle.com/schema/gradle-enterprise-maven.xsd">
+    <server>
+        <url>https://e.grdev.net</url>
+    </server>
+
+    <buildScan>
+        <publish>ALWAYS</publish>
+        <termsOfService>
```

Comment:
I suppose `termsOfService` only necessary for public `scans.gradle.com` - if we are using our own GE instance, we don't need this?

Reply:
I'll remove it for now.  I can restore it if we really want to use the public server.